server/datastore/cached_mysql/cached_mysql.go

```diff
@@ -95,7 +95,7 @@ func (ds *cachedMysql) AppConfig(ctx context.Context) (*fleet.AppConfig, error)
 func (ds *cachedMysql) SaveAppConfig(ctx context.Context, info *fleet.AppConfig) error {
 	err := ds.Datastore.SaveAppConfig(ctx, info)
 	if err != nil {
-		return errors.Wrap(err, "calling save app config")
+		return fmt.Errorf("calling save app config: %w", err)
```

Comment:
Why is this changed? Shouldn't we use `ctxerr.Wrap`?

Reply:
Or actually, just return err since it should already have the data it needs.

Reply:
Good catch, I think I was on a roll here :) It's true that the error from `Datastore.SaveAppConfig` should already have all it needs.